include/ebpf_program_types.h

```diff
@@ -87,7 +89,7 @@ typedef struct _ebpf_program_data
     ebpf_program_context_create_t context_create;   ///< Pointer to context create function.
     ebpf_program_context_destroy_t context_destroy; ///< Pointer to context destroy function.
     uint8_t required_irql;                          ///< IRQL at which the program is invoked.
-    bool context_header;                            ///< Whether the program context supports a context header.
+    uint32_t capabilities;                          ///< Capabilities supported by the program information provider.
```

Comment:
Use a bitfield rather than `#define`s, for compile type safety.  defines are much more bug prone.
It's fine to union it with `uint32_t capabilities` if there's a reason to do so but line 12 is not recommended.

Reply:
Updated to use bitfield.